articles/modules/ROOT/pages/how-to-force-match-ordering.adoc

```diff
@@ -0,0 +1,107 @@
+= How to Force Match Ordering
+:slug: how-to-force-match-ordering
+:author: Andrew Bowman
+:neo4j-versions: 3.5, 4.0, 4.1, 4.2, 4.3, 4.4, 5.x
+:tags: cypher
+:category: cypher
+
+Like SQL, Cypher is a declarative query language. This is most evident with Match patterns, which describe what you want to find in the graph.
+You do not dictate to it how to find these patterns, the way you might in an imperative programming language.
+
+Not only does the query planner decide how it will fulfill a single Match pattern, it has the ability to consider the entirety of patterns connected by common variables across multiple Match clauses in your query.
```

Comment:
Personally I like the `MATCH` and `WITH` and `WHERE` in backticks more as with and where are common English terms and it makes it more obvious than just capitalization that you mean the keywords.

Reply:
Good call, I'll make the change.